src/ray/protobuf/experimental/autoscaler.proto

```diff
@@ -65,14 +65,9 @@ message GangResourceRequest {
 // Cluster resource constraint represents minimial cluster size requirement,
 // this is issued through ray.autoscaler.sdk.request_resources.
 message ClusterResourceConstraint {
-  // If not empty, the cluster should have the capacity (total resource) to
-  // fit the min_resources.
-  map<string, double> min_resources = 1;
```

Comment:
I don't think we have such complicated strucutre for requested resources, 

see here at the client side: 
https://github.com/ray-project/ray/blob/9b45eb30fd9b033ab754bb8fb16f7822b560fc19/python/ray/autoscaler/_private/commands.py#L174-L184

Reply:
nice, we can remove it!